cli/cdsctl/template.go

```diff
@@ -0,0 +1,214 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ovh/cds/cli"
+)
+
+var (
+	templateCmd = cli.Command{
+		Name:  "template",
+		Short: "Manage CDS workflow template",
+	}
+
+	template = cli.NewCommand(templateCmd, nil, []*cobra.Command{
+		cli.NewCommand(templateApplyCmd, templateApplyRun, nil, withAllCommandModifiers()...),
```

Comment:
i think templateApplyCmd should be accessible also from `cdsctl workflow ...` may be in `cdsctl workflow init`

Reply:
done, applyTemplate